cmd/root.go

```diff
@@ -37,14 +35,14 @@ var rootCmd = &cobra.Command{
 		}
 
 		runner.ExecutePolicies()
-		results := runner.GetResults()
+		// results := runner.GetResults()
 
-		formattedResults, err := formatters.GenericJSONFormatter(results)
-		if err != nil {
-			return err
-		}
+		// // formattedResults, err := formatters.GenericJSONFormatter(results)
+		// // if err != nil {
+		// // 	return err
+		// // }
 
-		fmt.Fprint(os.Stdout, string(formattedResults))
+		// // fmt.Fprint(os.Stdout, string(formattedResults))
```

Comment:
We need to wire the output formatting back in. There's a separate PR that actually makes it dynamic based on user configuration, so returning it back to the state it was in should work.